papers/static/papers/refresh.js

```diff
@@ -68,6 +68,10 @@ function init_paper_module (config) {
       return gettext('No title')
     } else if (reason === 'INVALID_PUB_DATE') {
       return gettext('Invalid publication date')
+    } else if (reason === 'NO_PUBDATE') {
+      return gettext('No publication date')
```

Comment:
Should I add this string somewhere in the `.po` files?

Reply:
This should be handled automatically by `./manage.py makemessages -l fr`. We should run this regularly to update the strings to translate in TranslateWiki.